spynl/main/error_views.py

```diff
@@ -52,7 +52,8 @@ def error400(exc, request):
                 default="Permission to '${permission}' ${context} was denied.",
                 mapping={'context': request.context.__class__.__name__,
                          'permission': exc.result.permission})
-            emeta = exc.result  # TODO: log this as detail info
+            # TODO: log this as detail info
+            # emeta = exc.result
```

Comment:
Does anyone know how this is preferred to be logged so I can do it and update the PR?